src/Behat/MinkExtension/Context/MinkContext.php

```diff
@@ -54,7 +64,10 @@ public function reload()
     }
 
     /**
-     * Moves backward one page in history.
+     * Moves backward one page in history
+     * Example: Given I move backward one page
+     * Example: When I move backward one page
+     * Example: And I move backward one page
```

Comment:
A single example would be enough here (Given is wrong, and ``and`` is useless as it is exactly the same)